packages/squiggle-lang/src/rescript/Reducer/Reducer_Dispatch/Reducer_Dispatch_BuiltIn.res

```diff
@@ -147,6 +191,69 @@ let callInternal = (call: functionCall, environment, reducer: ExpressionT.reduce
     )
   }
 
+  let typeModifier_memberOf = (aType, anArray) => {
```

Comment:
small note, but I've been liking modules to organize code, instead of extra naming. Like,
```
module TypeModifiers = {
  let memberOf = ...
}
```

Reply:
I am delegating this to "lambda bindings for all internal functions" issue. The current code heap is about to disappear.

Because when I remove the switch statement, everything will only exist class by class.. We will have such modules
Library/Array
Library/String
Library/Number
Library/TypeIdentifier
etc